packages/bpk-component-graphic-promotion/src/BpkGraphicPromo.module.scss

```diff
@@ -31,6 +31,42 @@
   height: 100%;
 }
 
+@mixin content-alignment {
+  &--start {
+    align-items: start;
+  }
+
+  &--center {
+    align-items: center;
+  }
+
+  &--end {
+    align-items: end;
+  }
+}
+
+@mixin text-alignment {
```

Comment:
Extracted to fix a bug caused by the Style-Lint rules that move this automatically relative to the other elements in the page. This was causing the styles to be overridden in the wrong order.